tests/CsvFileErrorsTest.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace Keboola\Csv\Tests;
+
+use Keboola\Csv\CsvFile;
+use Keboola\Csv\Exception;
+use Keboola\Csv\InvalidArgumentException;
+use PHPUnit\Framework\TestCase;
+
+class CsvFileErrorsTest extends TestCase
+{
+    public function testNonExistentFile()
```

Comment:
testuje prakticky to samé co test pod tím IMO

Reply:
Jo vypadjí uplně stejně